packages/common/tests/customChains.spec.ts

```diff
@@ -123,19 +122,19 @@ tape('[Common]: Custom chains', function (t: tape.Test) {
       customChains: [testnet],
     })
     st.equal(c.chainName(), 'mainnet', 'customChains, chain set to supported chain')
-    st.ok(c.hardforkBlock()!.eqn(4370000), 'customChains, chain set to supported chain')
+    st.ok(c.hardforkBlock()! === BigInt(4370000), 'customChains, chain set to supported chain')
 
     c.setChain('testnet')
     st.equal(c.chainName(), 'testnet', 'customChains, chain switched to custom chain')
-    st.ok(c.hardforkBlock()!.eqn(4), 'customChains, chain switched to custom chain')
+    st.ok(c.hardforkBlock()! === BigInt(4), 'customChains, chain switched to custom chain')
```

Comment:
Just to mention here, also for @acolytec3: we *can* very well use the *n notation in the tests, the limitation to the BigInt() constructor is just for the production code (correct me if I am missing something).

Reply:
(of course also no need to do or even change here - again - just as some note)